frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -16,7 +16,7 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
     return (
       <div className={classNames('co-status-card__health-item', className)}>
         {state === HealthState.LOADING ? (
-          <div className="skeleton-health" />
+          <div className="skeleton-health" aria-label={`Loading ${title} Status`} />
```

Comment:
I'm not sure about using `aria-label` on a `div`. @jessiehuff Do you know if this works?

Reply:
Based on https://www.w3.org/WAI/GL/wiki/Using_aria-label_to_provide_labels_for_objects it looks that it can, though it should also include a `role`. I updated the pr to include `role="status` since it is a identifying a transient state - more info https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22.html

`<div className="skeleton-health" aria-label={`Loading ${title} Status`} role="status" />`

Reply:
I don't really recommend using an `aria-label` on a `div`. Leonie Watson, who is a prominent thought leader in the accessibility community, wrote up this [note](https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/) on labeling usage with different elements, and using aria labels on a div are not likely to work across all browser/assistive technology combinations. What we could do instead is use the [screen reader utility](https://www.patternfly.org/v4/documentation/core/utilities/accessibility#screen-reader-only) to add text specifically for screen reader users.

Reply:
I have updated the pr to use `.pf-u-screen-reader` with accessibility text associated.